packages/shortcuts-extension/schema/plugin.json

```diff
@@ -207,7 +207,7 @@
       "properties": {
         "command": { "default": "notebook:change-cell-to-code" },
         "keys": { "default": ["Y"] },
-        "selector": { "default": ".jp-Notebook:focus" }
+        "selector": { "default": ".jp-Notebook.jp-mod-focus" }
```

Comment:
The changes in this file I think cause the keyboard shortcuts to work when the *editor* is focused, i.e., in edit mode, which is not what we want. I think the changes in this file should be reverted.

Reply:
See #3552.